platform/jobs/edxPlatformPythonUnitTestsPr.groovy

```diff
@@ -114,18 +158,18 @@ secretMap.each { jobConfigs ->
             }
            git {
                 remote {
-                    url(JENKINS_PUBLIC_GITHUB_BASEURL + jobConfig['platformUrl'] + '.git')
+                    url("https://github.com/edx/${jobConfig.repoName}.git")
                     refspec('+refs/pull/*:refs/remotes/origin/pr/*')
                     if (!jobConfig['open'].toBoolean()) {
-                        credentials(jobConfig['platformCredential'])
+                        credentials("${GITHUB_BOT_CREDENTIALS}")
                     }
                 }
                 branch('\${ghprbActualCommit}')
                 browser()
                 extensions {
-                    relativeTargetDirectory(jobConfig['repoName'])
+                    relativeTargetDirectory(jobConfig.repoName)
                     cloneOptions {
-                        reference("\$HOME/" + jobConfig['platformCloneReference'])
+                        reference("\$HOME/${jobConfig.repoName}/.git")
```

Comment:
Is the "/" needed before .git?

Reply:
tbh not sure, but I am just configuring it the same as it is on the current jobs

Reply:
For background, the purpose of the shallow clone is so that the checkout of the edx-platform repo at the SHA to be tested will not start with an initial clone, but instead start with objects at a point from which the changeset will be smaller and faster.

* I just ssh'd into a jenkins_worker and looked at /home/jenkins
  * /home/jenkins/edx-platform-clone and it looks like a clone from Jul 26 (which I'm guessing is when the current AMI that we are using was created.). 
* Seems like the right way to fix this line is to use $HOME/edx-platform-clone/.git. 
  * Note that this should also be true for the private job case, UNLIKE HOW IT IS CONFIGURED NOW.
  * Searching in the jenkins secrets in the secure repo, $HOME/edx-platform-private-clone/.git is being used by the private jobs now. This doesn't make sense, as that directory does not exist on the jenkins worker, and so using that directory as a reference won't save anybody anything.
* FWIW I'm not sure why it's $HOME/edx-platform-clone/.git rather than $HOME/edx-platform-clone. From my testing, they work equivalently. However the latter seems more straightforward, and is supported by the example in the git-clone documentation. :woman_shrugging: